modules/distribution/resources/api_templates/websub_api_template.xml

```diff
@@ -0,0 +1,80 @@
+                                            <header name="To" expression="$ctx:SUBSCRIBER_CALLBACK"/>
+                                            <property name="signature" expression="fn:concat('sha1=', hmac-generate($ctx:ORIGINAL_PAYLOAD, $ctx:SUBSCRIBER_SECRET))"/>
+                                            <property name="X-Hub-Signature" expression="$ctx:signature" scope="transport"/>
+                                            <call blocking="true">
+                                                <endpoint>
+                                                    <default />
+                                                </endpoint>
+                                            </call>
+                                            <class name="org.wso2.carbon.apimgt.gateway.mediators.webhooks.DeliveryDataMediator"/>
+                                        </sequence>
+                                    </target>
+                                </clone>
+                            </sequence>
+                        </target>
+                    </clone>
+                </then>
+                <else>
+                    <drop/>
+                </else>
+            </filter>
+        </inSequence>
+    </resource>
+    <resource methods="POST" url-mapping="/*">
+        <inSequence>
+            <clone sequential="true">
+                <target>
+                    <sequence>
+                        <class name="org.wso2.carbon.apimgt.gateway.mediators.webhooks.SubscribersPersistMediator"/>
+                        <property name="NO_ENTITY_BODY" value="true" scope="axis2" type="BOOLEAN" />
+                        <respond/>
+                    </sequence>
+                </target>
+                <target>
+                    <sequence>
+                        <property name="SUBSCRIBER_CALLBACK" expression="$url:hub.callback"/>
+                        <property name="SUBSCRIBER_TOPIC" expression="$url:hub.topic"/>
+                        <property name="SUBSCRIBER_APPLICATION_ID" expression="$ctx:api.ut.application.id"/>
+                        <header name="To" expression="$ctx:SUBSCRIBER_CALLBACK"/>
+                        <call>
+                            <endpoint>
+                                <default />
+                            </endpoint>
+                        </call>
+                        <class name="org.wso2.carbon.apimgt.gateway.mediators.webhooks.DeliveryDataMediator"/>
+                        <drop/>
+                    </sequence>
+                </target>
+            </clone>
+        </inSequence>
+    </resource>
+    <handlers>
+        <handler class="org.wso2.carbon.apimgt.gateway.handlers.streaming.webhook.WebhookApiHandler">
+            <property name="subscriptionResourcePath" value="/subscription"/>
+            <property name="topicQueryParamName" value="hub.secret"/>
+        </handler>
```

Comment:
this need to have authentication handler as well, @chanikag  ?

Reply:
yes in addition to the rest of handlers we need to include this.

Reply:
I think we might need cors as well, @chanikag ?

Reply:
yeah all other handlers are applicable.

Reply:
Addressed in https://github.com/wso2/product-apim/pull/9885/commits/050bf78f67bc9c2951e815d07c2777fcd80bb627